scripts/reformat_data_hospital.py

```diff
@@ -99,7 +99,9 @@ def main():
               index=False,
               date_format='%Y-%m-%d',
               )
-    os.system("sudo cp " + FILENAME_REFORMATED + ' /usr/share/')
+
+    print('copy ' + FILENAME_REFORMATED + ' to /usr/share/ with:')
+    print("sudo cp " + FILENAME_REFORMATED + ' /usr/share/.')
```

Comment:
:-D

On Mon, May 2, 2016 at 4:04 PM, Andrew Rich <notifications@github.com>
wrote:

> In scripts/reformat_data_hospital.py
> <https://github.com/arich/hospital_readmission_server/pull/8#discussion_r61804183>
> :
>
> > @@ -99,7 +99,9 @@ def main():
> >                index=False,
> >                date_format='%Y-%m-%d',
> >                )
> > -    os.system("sudo cp " + FILENAME_REFORMATED + ' /usr/share/')
> > +
> > +    print('copy ' + FILENAME_REFORMATED + ' to /usr/share/ with:')
> > +    print("sudo cp " + FILENAME_REFORMATED + ' /usr/share/.')
>
> Noice.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/arich/hospital_readmission_server/pull/8/files/9358d13ab57c02f85959417e0eec9c8b180b2c3e..8e9d6e7ce862c6647f3dc6f938d9481e67eb9072#r61804183>
>



-- 
Elijah Bernstein-Cooper

Visit my website: ezbc.me